tests/phpunit/elementor/app/import-export/test-usage.php

```diff
@@ -0,0 +1,61 @@
+<?php
+namespace Elementor\Tests\Phpunit\Elementor\App\ImportExport;
+
+use Elementor\App\Modules\ImportExport\Usage;
+use ElementorEditorTesting\Elementor_Test_Base;
+
+class Test_Usage extends Elementor_Test_Base {
+
+	public function setUp()
+	{
+		parent::setUp();
+
+		remove_all_filters('elementor/tracker/send_tracking_data_params');
+
+		( new Usage() )->set_usage_data();
+	}
+
+	public function test_get_usage_data__without_any_data()	{
+		// Act.
+		$usage = apply_filters('elementor/tracker/send_tracking_data_params', []);
```

Comment:
```suggestion
		$usage = apply_filters('elementor/tracker/send_tracking_data_params', []);
```

PHPCS this whole file

Reply:
![giphy](https://user-images.githubusercontent.com/95690086/189588711-48530de2-1a55-40af-be2a-250766098cf6.gif)